Formula/psqlodbc.rb

```diff
@@ -19,13 +19,19 @@ class Psqlodbc < Formula
   end
 
   depends_on "openssl"
-  depends_on "unixodbc"
   depends_on :postgresql
+  depends_on "libiodbc" => :optional
+  depends_on "unixodbc" if build.without? "libiodbc"
```

Comment:
I did not do this originally because `unixodbc` and `libiodbc` are mutually exclusive. They cannot both be linked at the same time. So in order to use `libiodbc` you need to specify `--with-libiodbc --without-unixodbc`. But if you say the `if build.with?` conditionals don't work well I guess there is no choice.